testrunner/sa.tf

```diff
@@ -33,3 +37,63 @@ resource "google_project_iam_binding" "storage_admin" {
     "serviceAccount:${google_service_account.testrunner_service_account[0].email}"
```

Comment:
You can't use [google_project_iam_binding](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_project_iam#google_project_iam_binding) here, because it is an authoritative resource declaration. When you ran `atlantis plan -p testrunner-alpha`, it changed the member list for `roles/storage.admin` to contain only your testrunner SA.

To fix this, you need to use the  [google_project_iam_member](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_project_iam#google_project_iam_member) Terraform resource to modify the membership list for `roles/storage.admin` to add your testrunner SA.

Reply:
I have updated to use  `google_project_iam_member`. There are other `google_project_iam_binding` resources, should I change them too?